src/main/java/com/sergiofah/model/Line.java

```diff
@@ -24,4 +25,15 @@ public class Line {
     @OneToMany(mappedBy = "line")
     private List<Category> categories;
 
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) return true;
+        if (object == null) return false;
+
+        if(object.getClass() == LineDTO.class){
+            LineDTO lineDTO = (LineDTO) object;
+            return this.id.equals(lineDTO.getId()) && (this.line.equals(lineDTO.getLine()));
+        }
```

Comment:
Favor, não usar o operador "==" quando for comparar dois objetos, apenas quando for números. Isso porque o operador "==" mesmo sendo mais rápido (porém não é notório) ele não tem uma assertividade tão boa quanto usando o método equals(). Isso pode até ser explicado para o caso de sobrescrever o método equals(), imagine caso o método o assertArrayEquals usasse o operador "==" ao invés do método equals dos objetos que ele recebe como parâmetro.